src/components/KeyboardShortcuts/index.spec.tsx

```diff
@@ -2,12 +2,20 @@ import * as React from 'react';
 import { Store } from 'redux';
 
 import configureStore from '../../configureStore';
-import { RelativePathPosition } from '../../reducers/fileTree';
-import { actions as versionActions } from '../../reducers/versions';
+import {
+  actions as fileTreeActions,
+  RelativePathPosition,
+} from '../../reducers/fileTree';
+import {
+  actions as versionActions,
```

Comment:
I know this is a carryover from an error of mine, but I think we want to call this `versionsActions` not `versionActions`.